KLEE: Using Z3 solver backend
KLEE: ERROR: /home/walker/Projects/Experiment/1/case/float_extension/float_extension_i.cc:26: failed external call: _ZNSolsEPFRSt8ios_baseS0_E
KLEE: NOTE: now ignoring this error at this location
